src/applications/disability-benefits/all-claims/tests/disability-benefits-helpers.js

```diff
@@ -14,9 +14,9 @@ function initDocumentUploadMock() {
   });
 }
 
-function initApplicationSubmitMock() {
-  mock(null, {
-    path: '/v0/21-526EZ',
+function initApplicationSubmitMock(token) {
+  mock(token, {
```

Comment:
Mocking the endpoint without the user token meant that when we hit it while authenticated, this mock data wouldn't get returned. Thanks, @bshyong, for figuring this out! :raised_hands: